sites/partners/src/components/listings/PaperListingForm/index.tsx

```diff
@@ -66,6 +66,7 @@ type ListingFormProps = {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const ListingForm = ({ listing, editMode }: ListingFormProps) => {
   const defaultValues = editMode ? listing : formDefaults
+  const isListingOpen = listing?.status === "active"
```

Comment:
@KrissDrawing It looks like there's another place in this file where the listing status is checked: `listing?.status === ListingsStatusEnum.active`

Since you're adding this, can you update the other one? Also I'm wondering if `"active"` should become `ListingStatusEnum.active`